ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -392,25 +414,33 @@ translateLedgerStateByronToShelleyWrapper ::
      )
   => RequiringBoth
        WrapLedgerConfig
-       (Translate LedgerState)
+       TranslateLedgerState
        ByronBlock
        (ShelleyBlock (TPraos c) (ShelleyEra c))
 translateLedgerStateByronToShelleyWrapper =
```

Comment:
The documentation of `translateLedgerStateWith` in the `TranslateLedgerState` datatype says that there are two cases where it is important that `translateLedgerStateWith` returns `DiffMK`. However, in this translation, the ledger tables are empty! Is this comment outdated?

Reply:
No they are not, they have diffs